src/components/about/AboutPage.js

```diff
@@ -1,17 +1,25 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { PreferenceDemo } from "../../redux/dispatch/Demo/PreferenceDemo";
 
-const AboutPage = () => (
-  <div>
-    <h2>About</h2>
-    <p>
-      This application uses React, Redux, Redux thunk, React context provider for global data management,
-      React Router, and many other helpful libraries. In addition, we are using ThemeProvider and styled components
-      to build a theme of your choice and apply it to the rest of application on top of predefined sets of themes.
+const AboutPage = () => {
 
-      Typescript is deployed to render all queued albums related pages, so coexistence of Babel and Typescript loaders
-      is a feature of the entire application.
-    </p>
-  </div>
-);
+    useEffect(() => {
+        PreferenceDemo();
+    }, []);
```

Comment:
^ this is the only new bits in this file. The rest of it got reformatted by Visual Studio, sorry!

Reply:
Lovely! I can compile and run now without any errors, will use this DispatchBuilder to dispatch song or singer related actions.